README.md

```diff
@@ -24,116 +23,203 @@
 [![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
 [![Open Source Helpers](https://www.codetriage.com/plasmapy/plasmapy/badges/users.svg)](https://www.codetriage.com/plasmapy/plasmapy)
 
-[PlasmaPy](https://www.plasmapy.org/) is an open source, community-developed
-Python 3.8+ package for plasma science. PlasmaPy intends to be for plasma
-science what [Astropy](https://github.com/astropy/astropy) is for astronomy
-— a collection of functionality commonly used and shared between plasma
-scientists and researchers globally, running within and leveraging the
-open source scientific Python ecosystem. The goals of this project are
-more thoroughly described in [this recent video](https://youtu.be/E8RwQF5wcXM).
-Current functionality is described in [PlasmaPy's online
-documentation](http://docs.plasmapy.org/en/latest/).
-
-We created a guide on [contributing to PlasmaPy](http://docs.plasmapy.org/en/stable/CONTRIBUTING.html)
-and have a [code of conduct](http://docs.plasmapy.org/en/stable/CODE_OF_CONDUCT.html).
-New contributors are very welcome!
-
-# Installation
-
-If you have [installed Python](https://wiki.python.org/moin/BeginnersGuide/Download),
-you can install PlasmaPy from [pip](https://pypi.org/project/pip/)
-via
+[Anaconda Navigator]: https://docs.anaconda.com/anaconda/navigator/index.html
+[Astropy]: https://www.astropy.org
+[calendar]: https://calendar.google.com/calendar/embed?src=c_sqqq390s24jjfjp3q86pv41pi8%40group.calendar.google.com&ctz=America%2FNew_York
+[citation instructions]: https://docs.plasmapy.org/en/latest/about/citation.html
```

Comment:
I get the intention behind splitting the links off, but... I honestly think for short links and for one off links, keeping them in the main text body makes for a much cleaner writing experience. I'd say it's fine to keep repetitive abominations such as the extremely long calendar link in the above, though. I'll go through this and propose some adjustments in commit form.